### PR TITLE
Upgrade kotlintest -> kotest

### PR DESCRIPTION
**Pull request by bbaldino**

### CODE REVIEW

File: jvb/pom.xml [hunk 1]
```diff
@@ -17,6 +17,7 @@
   <url>https://github.com/jitsi/jitsi-videobridge</url>
   <properties>
     <assembly.skipAssembly>false</assembly.skipAssembly>
+    <kotest.version>4.1.2</kotest.version>
```
Comment (bgrozev) [inline jvb/pom.xml:20]: Any reason not to use 4.1.3?

File: jvb/pom.xml [hunk 2]
```diff
@@ -285,12 +286,26 @@
...
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>io.kotlintest</groupId>
-      <artifactId>kotlintest-runner-junit5</artifactId>
-      <version>3.4.2</version>
-      <scope>test</scope>
-    </dependency>
+	<dependency>
+		<groupId>io.kotest</groupId>
```
Comment (bgrozev) [inline jvb/pom.xml:290]: The rest of the file uses 2 spaces for indentation

File: jvb/pom.xml (hunk 1 above)
Comment (bbaldino) [reply jvb/pom.xml:20]: I looked [here](https://github.com/kotest/kotest) and saw latest release as 4.1.2 so used that...I'll change to 4.1.3

File: jvb/pom.xml (hunk 2 above)
Comment (bbaldino) [reply jvb/pom.xml:290]: Tab snuck in there.

**Pull request merged**